docs/src/main/asciidoc/spring-cloud-netflix.adoc

```diff
@@ -2115,11 +2115,41 @@ class MyFallbackProvider implements FallbackProvider {
 
 === Zuul Timeouts
 
-If you want to configure the socket timeouts and read timeouts for requests proxied through
-Zuul there are two options based on your configuration.
+==== Service Discovery Configuration
 
-If Zuul is using service discovery than you need to configure these timeouts via Ribbon properties,
-`ribbon.ReadTimeout` and `ribbon.SocketTimeout`.
+If Zuul is using service discovery there are two timeouts you need to be concerned
+with, the Hystrix timeout (since all routes are wrapped in Hystrix commands by default)
+and the Ribbon timeout.  The Hystrix timeout needs to take into account the Ribbon
+read and connect timeout PLUS the total number of retries that will happen for that
+service.  By default Spring Cloud Zuul will do its best to calculate the Hystrix timeout
+for you *UNLESS* you specify the Hystrix timeout explicitly.
+
+The Hystrix timeout is calculated using the following formula:
+`(ribbon.ConnectTimeout + ribbon.ReadTimeout) * (ribbon.MaxAutoRetries + 1) * (ribbon.MaxAutoRetriesNextServer + 1)`
```

Comment:
Lets surround this in triple ticks (```) so it is placed on its own line

Reply:
My fault, I am not careful enough. I have already edited it.